### DISCUSSION TITLE
Tailwind CSS IntelliSense stuck [Global] Creating projects: []

### DISCUSSION THREAD

**Discussion by lucifer** [tailwindlabs/tailwindcss#10727] [Help] [answered]
I have a perfectly formed tailwind.config.js file: -
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
 and re-installed the vscode extension, restarted the editor. 
VSCode output for Tailwind CSS IntelliSense is a single line: -
```
[Global] Creating projects: []
```
After hours searching for the oh so many autocomplete issues with this extension, I'm out of wits, hoping I uncovered something we can fix easily to help others.

**Comment by lucifer** [comment, marked as answer]
Changed tailwind.config.js to this: -
```
module.exports = {
  content: [
    './**/*.{html,js}',
  ],
}
```

Now it works, why though? I followed official docs here: -
https://tailwindcss.com/docs/installation

**Comment by onionhammer** [comment]
I'm having the same issue.. not sure why, all of the sudden.

I've tried uninstalling and reinstalling the plugin

**Comment by khaledYS** [comment]
Same Issue unfortunately, I hope we find a sollution.
The above accepted sollution didn't work. @lucifer

**Comment by TillHeinzel** [comment]
I have had the same problem repeatedly. 

It seems like the plugin checks for tailwind files inside the workspace, not the folder on disc. I use a "scope-to" extension, which uses vs code exclude functionality. So it couldn't find the tailwind configfile, because it was excluded. Bit annoying, but apparently, I can't use these two plugins together. Basically, a possible issue may be that the actual configfile can't be found inside the workspace due to being hidden or excluded in settings.

**Comment by Der-Alex** [comment]
Some questions: 
- Do you have a ./src folder?
- Do you use some frontend framework (vue, react, etc...) ?
- Do you use other tools in your dev environment like vite, typescript, etc?

It looks like something is misconfigured, that makes tailwind not find the ./src folder
